doc/services.rst

```diff
@@ -70,11 +70,11 @@ Amazon EMR
 .. autoclass:: stepfunctions.steps.service.EmrModifyInstanceGroupByNameStep
 
 Amazon EventBridge
------------
+------------------
```

Comment:
curious: is this visible to customers or is it just normalizing the style?

Reply:
This addresses some warnings that came up when generating the documentation locally - i can include this in a separate PR together with the changes to fix the doc links